pydm/widgets/timeplot.py

```diff
@@ -348,9 +525,21 @@ def channels(self):
                          BasePlot.setMaxYRange, doc="""
     Maximum Y-axis value visible on the plot.""")
 
+    def getShowRightAxis(self):
+        return self._show_right_axis
 
-class TimeAxisItem(AxisItem):
+    def setShowRightAxis(self, show):
+        self._show_right_axis = show
```

Comment:
Shouldn't this also trigger the display of the Axis on the right?

Reply:
But it is only setting a boolean variable... where is it checking or doing the real code to show the right axis?

Reply:
You're right, this needs the actual API call to set the right y-axis. I didn't actually call this method for the charting tool, so I didn't notice it. Fixed.